src/providers/object-provider.js

```diff
@@ -176,44 +176,40 @@ export default class YamcsObjectProvider {
     async #getTelemetryDictionary() {
         if (this.dictionaryLoaded) {
             return this.dictionary;
```

Comment:
Where is this.dictionary being assigned? Maybe missed that?

Reply:
https://github.com/akhenry/openmct-yamcs/blob/dac68ba6d18f9769fa4d971bf7d84476386551c2/src/providers/object-provider.js#L209 this triggers that.